lib/feature.rb

```diff
@@ -0,0 +1,3 @@
+require 'feature_config/feature'
+
+Feature = FeatureConfig::Feature
```

Comment:
I think we shouldn't do this. I made this to make the current code sync with previous implementation.